### ISSUE TITLE
The quote style changes to normal after Applying Styles

### ISSUE THREAD

**Issue by actforjason** [stevencohn/OneMore#705]
The custom style is placed in the `style=""`, the style is reset to `p` according to `quickstyleindex = "1"` after Applying Styles to page. The correct way to do this is to record the custom style as another `QuickStyleDef `

![image](https://user-images.githubusercontent.com/57058325/199931043-9dcaada3-ee6f-4129-b90a-de730a3038d2.png)

**Comment by stevencohn** [comment]
Working as intended. Apply Styles to Page is intended to wipe out any inline customizations.

**Comment by actforjason** [comment]
But Apply Styles to Page also cleares Onemore's own styles. And what users want is to replace OneNote's own styles with onemore's

**Comment by actforjason** [comment]
Do you mean that the style of Onemore can only be applied on the basis of the OneNote style, instead of being used alone (because it is always reset)?

**Comment by actforjason** [comment]
In this way, for the purpose of maintaining the style，there must be two steps：First apply the OneNote style, and then use the OneMore style cover it.

**Comment by actforjason** [comment]
For OneMore, its styles H1 ~ H6 can create QuickStyleDef to maintain the style, and other others, such as citation, quote will disappear once the style is applied to the page.

**Comment by stevencohn** [comment]
No. OneMore will create its own quickstyles. But OneNote dictates that some attributes are specified by the quickstyle and some attributes are applied inline. This is how OneNote works.

For example, look at the XML of a OneNote H4 heading:

```xml
      <one:OE alignment="left" quickStyleIndex="1" selected="partial">
        <one:T><![CDATA[<span style='font-style:italic'>Example</span>]]></one:T>
        <one:T selected="all"><![CDATA[]]></one:T>
      </one:OE>
```

Even though the quickstyle specifies italic

```xml
  <one:QuickStyleDef index="1" name="h4" fontColor="#5B9BD5" highlightColor="automatic" font="Calibri" fontSize="12.0" italic="true" spaceBefore="0.0" spaceAfter="0.0" />
```

It still adds the font-style attribute inline.

**Comment by stevencohn** [comment]
> For OneMore, its styles H1 ~ H6 can create QuickStyleDef to maintain the style, and other others, such as citation, quote will disappear once the style is applied to the page.

Not sure what you mean. If there is a OneNote "Citation" on the page, and there is a OneMore "Citation" definition, then the OneMore definition will override it. 

But if there is NOT a OneMore "Citation" definition then it will leave it alone, effectively keeping it as a OneNote Citation

**Comment by actforjason** [comment]
OneNote's "Citation" will create QuickStyle `blockquote,` but OneMore's only `style=` and its parent is `p` that will cover `style=` after applying style to page.
![image](https://user-images.githubusercontent.com/57058325/199987601-b5eb2d8d-c4a9-4151-989f-36378b94e0b9.png)
![image](https://user-images.githubusercontent.com/57058325/199987608-7a282f0e-ac8d-4d87-b42a-14122443128b.png)

**Comment by stevencohn** [comment]
OneNote's Citation quickstyle should have the name "cite", not "blockquote". Are you looking at the right thing?

And yes, the definition of OneMore's Citation should derive from "p". It can't call itself "cite" because that will conflict with OneNote's "cite"

I suggest you have a play with this all programmatically. Play around with the OneMore code or write your own to learn how this all works. There are hundreds of little scenarios that I've worked through over the years and I can't explain them all in a chat thread.

**Comment by actforjason** [comment]
However, OneMore's H1 to H6 quickstyle names are still the same as OneNote's. In fact, the OneMore style is a perfect alternative to OneNote, and using the two together leads to style confusion. This means that OneMore can simply define the same named quickstyle to override OneNote's inline styles to avoid confusion.

**Comment by actforjason** [comment]
> the definition of OneMore's Citation should derive from "p".
In this case, Citation will use the p style after applying style to page, while OneMore's style for Citation will be lost.

**Comment by stevenmcohn** [comment]
There are a number of style theme files installed with OneMore. Use the Load command to load other style definitions

**Comment by stevencohn** [comment]
There are a number of style theme files installed with OneMore. Use the Load command to load other style definitions

**Comment by actforjason** [comment]
No matter what the theme，OneMore's H# will override OneNote's inline quickstyle.
https://github.com/stevencohn/OneMore/blob/902cf770a9d3b7c435440a0cc254aea7222c043b/OneMore/Commands/Styles/ApplyStyleCommand.cs#L246-L260

**Comment by stevencohn** [comment]
I can't say this enough - this is the way it is supposed to work. It has to work this way. It can't work any other way. It is what it is. I don't know what else to say. Sorry.

**Comment by actforjason** [comment]
OK, thank you for your time. I'll try to change it for my own use.
